Source/EventFlow.ServiceProvider/EventFlow.ServiceProvider.csproj

```diff
@@ -0,0 +1,33 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <TreatWarningsAsErrors>True</TreatWarningsAsErrors>
+    <GenerateAssemblyInfo>True</GenerateAssemblyInfo>
+    <GeneratePackageOnBuild>False</GeneratePackageOnBuild>
+    <PackageRequireLicenseAcceptance>True</PackageRequireLicenseAcceptance>
+    <Title>EventFlow.ServiceProvider</Title>
+    <Authors>Rasmus Mikkelsen</Authors>
```

Comment:
@frankebersoll Please put your name in the `Authors`. I didn't write it and I don't deserve the credits.